indico/modules/rb/client/js/components/TimeRangePicker.jsx

```diff
@@ -26,9 +26,11 @@ function _humanizeDuration(duration) {
   if (hours === 1 && minutes === 0) {
     return Translate.string('1 hour');
   } else if (hours !== 0) {
-    return Translate.string('{time} hours', {time: hours + minutes / 60});
+    return PluralTranslate.string('{time} hour', '{time} hours', hours + minutes / 60, {
+      time: hours + minutes / 60,
+    });
```

Comment:
Can we combine this with the previous case which checks `hours === 1 && minutes === 0`?

```diff
- if (hours === 1 && minutes === 0) {
-   return Translate.string('1 hour');
- } else if (hours !== 0) {
+ if (hours !== 0) {
  return PluralTranslate.string('{time} hour', '{time} hours', hours + minutes / 60, {
    time: hours + minutes / 60,
  });
}
```

Reply:
Sure, did that. I was too focused on changing just a single line to avoid breaking stuff and didn't look at the context.